docs/UserGuide.md

```diff
@@ -91,25 +93,25 @@ A handy reference for more experienced users who just need to know the format of
 * Attributes can be in any order.<br>
   e.g. if the command asks for `n/NAME p/PHONE_NUMBER`, `p/PHONE_NUMBER n/NAME` is also fine.
 
-* Items in square brackets are optional.<br>
-  e.g. `n/NAME [t/TAG]` can be typed as `n/John Doe t/friend` or as `n/John Doe`.
-
-* Items with `...​` after them can be used multiple times including zero times.<br>
-  e.g. `[t/TAG]...​` can be used as `t/friend`, `t/friend t/family` or even 0 times.
-
 * If an attribute is expected only once, but you type it multiple times, only the last occurrence of the attribute will be taken.<br>
   e.g. if you specify `n/Jake n/Jason`, only `n/Jason` will be taken.
 
 * For commands that do not need attributes, (like `help`, `list`, `exit` and `clear`) anything typed after the command word will be ignored.<br>
   e.g. typing `help 123`, it will be interpreted as `help`.
 
+* Items in square brackets are optional.<br>
+  e.g. `n/NAME [t/TAG]` can be typed as `n/John Doe t/friend` or as `n/John Doe`.
+
+* Items with `...​` after them can be used multiple times or not at all.<br>
+  e.g. `[t/TAG]...​` can be used as `t/friend`, `t/friend t/family`
+
 </div>
 
 --------------------------------------------------------------------------------------------------------------------
 
 ## Employee-related commands
 
-### Add an employee : `add`
+### How to add an employee ? `personadd`
```

Comment:
I think the previous layout would look better in the table of contents? wdyt?

Reply:
additionally, if this were to be the preferred layout, the space before the question mark might look a bit odd

Reply:
This is based on a specific comment from Ms Richard I think. See #102. 

 Can revert to the original, if preferred. Accepting the edit on the space before the `?` though.